python-ecosys/iperf3/iperf3.py

```diff
@@ -147,10 +147,10 @@ def stop(self):
 
     def report_receiver(self, stats):
         st = stats["streams"][0]
-        dt = st["end_time"] - st["start_time"]
+        dt = ticks_diff(self.t3, self.t0)
```

Comment:
I think this line needs a comment, to mention that servers pre-3.2 don't send these values and so in this case the dt is computed as our (local) dt.

Reply:
Done in 2056dac